packages/types/src/interfaces/ethBridge/definitions.ts

```diff
@@ -26,6 +26,7 @@ export default {
       "eventClaimId": "EventClaimId",
       "authorityIndex": "u16",
       "result": "EventClaimResult"
-    }
+    },
+    details: 'Vec<(EthAddress, Vec<u8>, u8 )>',
```

Comment:
```suggestion
    Details: 'Vec<(EthAddress, Vec<u8>, u8 )>',
```
convention is **D**etails